docs/custom-tests/02-tagging-tests.md

```diff
@@ -0,0 +1,81 @@
+# Tagging Tests
+
+:::info
+The Custom Tests feature is coming soon.
+:::
+
+## Introduction
+
+You can use test tags to run specific tests from a plugin.
+
+Each plugin can have one or multiple test tags, and we also offer some generic tests that can be used by any extensions.
```

Comment:
Minor nitpick:

```suggestion
Each plugin can have one or multiple test tags, and we also offer some generic tests that can be used by any extension.

We also use extensions and plugins both on this page, should we maybe use `plugin` here instead? What do you think?
```

Reply:
Hmm, I'm not sure.... I don't like "extension" very much, but themes are theoretically supported, you just have to run them with `--testing_theme` (I plan to get rid of this requirement at a later point).

Reply:
That flag is required when the SUT is a theme

Reply:
Just pushed a commit, what do you think?

Reply:
I'm in the same boat, I tend to prefer plugin over extension generally myself.

Reply:
I'll add a dedicated page about themes with all the nitpicks

Reply:
> Just pushed a commit, what do you think?

Those tweaks look good to me!

Reply:
I've added a page about Themes as well

Reply:
> Those tweaks look good to me!

All right, so I'm merging away